CHANGELOG.md

```diff
@@ -1,6 +1,33 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.10.7:
+	- Added ability to avoid limiting robot agents (CFE-3161)
+	- Added and transitioned to using master_software_updates shortcut
+	  (ENT-4953)
+	- Added continual checking for policy_server state (CFE-3073)
+	- Added documentation how to enable systemd unit management and disable agents on all hosts
+	  (CFE-3416)
+	- Added package_module for snap (CFE-2811)
+	- Always set files_single_copy from augments if available (CFE-3064)
+	- Enabled systemd unit management by default (CFE-2429)
```

Comment:
No, we reverted this systemd unit management by defualt, it shoudl be redacted from the changelog.